api/utils/ApplicationCodeHelper.js

```diff
@@ -0,0 +1,22 @@
+const crypto = require('crypto');
+
+const generate = async () => {
+  let code = null;
+
+  let existingCodes = await MedicalReport.findAll({
+    attributes: ['applicationCode'],
+    raw: true
+  });
+
+  const codes = _.pluck(existingCodes, 'applicationCode');
```

Comment:
Future improvement: Switch to `_.map(...)`, with `_.pluck(...)` being deprecated in Lodash. Adding a Trello card for this.